doc/nrf/known_issues.rst

```diff
@@ -1183,33 +1184,33 @@ Bluetooth® LE
 
 .. rst-class:: v2-0-2
 
-DRGN-17695: The BT RX thread stack might overflow if the :kconfig:option:`CONFIG_BT_SMP` is enabled.
+DRGN-17695: The BT RX thread stack might overflow if the :kconfig:option:`CONFIG_BT_SMP` is enabled
   When performing SMP pairing MPU FAULTs may be triggered because the stack is not large enough.
 
   **Workaround:** Increase the stack size manually in the project configuration file (:file:`prj.conf`) using :kconfig:option:`CONFIG_BT_RX_STACK_SIZE`.
 
-.. rst-class:: v2-0-2 v2-0-1 v2-0-0
+.. rst-class:: v2-1-0 v2-0-2 v2-0-1 v2-0-0
```

Comment:
Fixed for 2.1.0 - please remove the new tag.

Reply:
Updated.